src/modules/account/api/get-user.api.ts

```diff
@@ -0,0 +1,15 @@
+import { apiClient } from '#/shared/libs/api-client.lib';
+import { type Profile } from '#/shared/types/profile.type';
+import { type WithBaseResponse } from '#/shared/types/with-base-response.type';
```

Comment:
Main branch doesn't have with-base-response anymore, only base-response. Please use ```BaseResponse```. Did you pull the latest changes?

Reply:
Fixed.